src/ol/control/ScaleLine.js

```diff
@@ -500,7 +460,7 @@ class ScaleLine extends Control {
     );
     const dpi = this.dpi_ || DEFAULT_DPI;
     const inchesPerMeter = 1000 / 25.4;
-    return parseFloat(resolution.toString()) * inchesPerMeter * dpi;
```

Comment:
If this did something useful I don't understand it.
It seems to convert the resolution from number to string to number?

Reply:
I agree it is not useful (oversight in review of #9013).